.rubocop.yml

```diff
@@ -7,8 +7,8 @@ AllCops:
   DisplayStyleGuide: true
   StyleGuideCopsOnly: false
   TargetRubyVersion: 2.5
-Layout/AlignArguments:
-  EnforcedStyle: with_fixed_indentation
+Layout/ArgumentAlignment:
+  Enabled: false
```

Comment:
Is there a reason for this?  Don't we like to indent with 2 spaces for argument alignment?

Reply:
Yeah, I was trying to get it to let me have 2 args on the first line. But i can change it back and just have 1 there.

Reply:
Hmm, I think that's gotta be a different rule, we do wrapping like this everywhere in kitsu don't we?

Reply:
Yeah I'm wondering if maybe the fact that we're using the wrong rule name on `the-future` is causing hound to complain about these. When I run rubocop locally it's all good.

Reply:
Ah yeah probably, I think hound only uses the master yml so you can't hide stuff in a PR by changing the rules

Reply:
Reverted this. And also fixed the rails rules by adding the rubocop-rails gem. Forgot I had left these commented there whoops. Is it okay if we merge this once approved, even with the outstanding hound messages, given that it passes local rubocop?